packages/template-retail-react-app/package.json

```diff
@@ -1,98 +1,74 @@
 {
-    "name": "retail-react-app",
-    "version": "2.8.0-dev",
-    "license": "See license in LICENSE",
-    "private": true,
-    "engines": {
-        "node": "^14.0.0 || ^16.0.0 || ^18.0.0",
-        "npm": "^6.14.4 || ^7.0.0 || ^8.0.0 || ^9.0.0"
-    },
-    "devDependencies": {
-        "@chakra-ui/icons": "^1.1.7",
-        "@chakra-ui/react": "^1.8.9",
-        "@chakra-ui/skip-nav": "^1.2.6",
-        "@chakra-ui/system": "^1.12.1",
-        "@emotion/react": "^11.10.6",
-        "@emotion/styled": "^11.10.6",
-        "@formatjs/cli": "^4.8.4",
-        "@lhci/cli": "^0.7.0",
-        "@loadable/component": "^5.15.3",
-        "@peculiar/webcrypto": "^1.4.2",
-        "@tanstack/react-query": "^4.0.10",
-        "@tanstack/react-query-devtools": "^4.22.0",
-        "@testing-library/dom": "^9.0.1",
-        "@testing-library/jest-dom": "^5.16.5",
-        "@testing-library/react": "^11.2.7",
-        "@testing-library/react-hooks": "^7.0.2",
-        "@testing-library/user-event": "^12.8.3",
-        "base64-arraybuffer": "^0.2.0",
-        "bundlesize2": "^0.0.31",
-        "card-validator": "^8.1.1",
-        "commerce-sdk-react-preview": "^2.8.0-dev",
-        "cross-env": "^5.2.1",
-        "cross-fetch": "^3.1.5",
-        "focus-visible": "^5.2.0",
-        "framer-motion": "^3.10.6",
-        "full-icu": "^1.5.0",
-        "helmet": "^4.6.0",
-        "jest-fetch-mock": "^2.1.2",
-        "js-cookie": "^3.0.1",
-        "jsonwebtoken": "^9.0.0",
-        "jwt-decode": "^3.1.2",
-        "lodash": "^4.17.21",
-        "msw": "^0.28.1",
-        "nanoid": "^3.3.4",
-        "njwt": "^1.2.0",
-        "prop-types": "^15.8.1",
-        "pwa-kit-dev": "2.8.0-dev",
-        "pwa-kit-react-sdk": "2.8.0-dev",
-        "pwa-kit-runtime": "2.8.0-dev",
-        "query-string": "^7.1.3",
-        "raf": "^3.4.1",
-        "randomstring": "^1.2.3",
-        "react": "^17.0.2",
-        "react-dom": "^17.0.2",
-        "react-helmet": "^6.1.0",
-        "react-hook-form": "^6.15.8",
-        "react-intl": "^5.25.1",
-        "react-router-dom": "^5.3.4"
-    },
-    "peerDependencies": {
-        "@chakra-ui/system": "^1.12.1"
-    },
-    "scripts": {
```

Comment:
Why are we dropping the npm scripts here?

Reply:
that's really weird, not intentional, I'll look

Reply:
fixed